examples/fastify-ssr/package.json

```diff
@@ -29,9 +28,11 @@
     "graphql-hooks-memcache": "^2.2.1",
     "graphql-hooks-ssr": "^2.2.1",
     "isomorphic-unfetch": "^3.0.0",
+    "prop-types": "^15.7.2",
     "react": "^17.0.0",
     "react-dom": "^17.0.0",
-    "react-tree-walker": "^4.3.0"
+    "react-router": "^6.0.2",
```

Comment:
react-router is not necessary anymore I believe (it used to in earlier versions)